assignments/array-methods.js

```diff
@@ -56,28 +56,90 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+
+runners.forEach(function(currentValue){
+    fullName.push(`${currentValue.first_name}, ${currentValue.last_name}`);
```

Comment:
not sure if this was intentional or not, but in a string literal, you dont want a comma, unless you want the comma as part of the string
```suggestion
    fullName.push(`${currentValue.first_name} ${currentValue.last_name}`);
```